bounties_api/user/views.py

```diff
@@ -47,6 +51,46 @@ def get(self, request):
         return HttpResponse('Unauthorized', status=401)
 
 
+class RequestProfileImageUploadURL(APIView):
+    permission_classes = [AuthenticationPermission]
+
+    def get(self, request):
+        AWS_REGION = 'us-east-1'
+        client = boto3.client('s3', region_name=AWS_REGION)
+
+        current_user = request.current_user
+        nonce = int(random() * 1000)
```

Comment:
this works... but there is also a uuid.uuidv4 library I use elsewhere :)

Reply:
good point, i switched it out :)